File: pingYouGou-manager-web/src/main/resources/static/js/service/sellerService.js
// 定义 Service
app.service("sellerService", function ($http, pingYouGouServer) {
    /**
     * 获取分页数据
     * @param page 当前页
     * @param size 每页数据
     */
    this.findPage = function (page, size) {
        let url = `${pingYouGouServer}/seller/findPage.do?pageNum=${page}&pageSize=${size}`;
        return $http.get(url, {
            pageNum: page,
            pageSize: size
        });
    }

    this.findSellerDetialInfo = function (sellerId) {
        let url = `${pingYouGouServer}/seller/findSellerDetialInfo.do?sellerId=${sellerId}`;
        return $http.get(url);
    }

    /**
     * 商户审核
     * @param sellerId
     * @param preStatus
     */
    this.checkSeller = function (sellerId, preStatus) {
        let status = undefined;
        if ("pass" == preStatus) {
            // 通过
            status = 1;
        } else if ("reject" == preStatus) {
            // 驳回
            status = 2;
        } else if ("close" == preStatus) {
            // 关闭
            status = 3;
        }
        let url = `${pingYouGouServer}/seller/sellerInfo.do/sellerId/${sellerId}/status/${status}`;
        return $http.put(url);

    }

})

File: pingYouGou-sellerGoods-interface/src/main/java/com/lemonzuo/sellergoods/service/SellerService.java
package com.lemonzuo.sellergoods.service;

import com.lemonzuo.model.PageResult;
import com.lemonzuo.model.Seller;

import java.util.List;

/**
 * @author LemonZuo
 * @create 2019-09-03 21:39
 */
public interface SellerService {
    /**
     * 新增商家信息
     * @param seller
     * @return
     */
    int addSeller(Seller seller);

    /**
     * 根据商家ID查询商家是否存在
     * @param sellerId
     * @return
     */
    Integer checkIsExist(String sellerId);

    /**
     * 带条件查询商家信息
     * @param seller
     * @return
     */
    PageResult<Seller> findSellerInfo(Seller seller, Integer pageNum, Integer pageSize);

    /**
     * 根据商家ID查询商家详细信息
     * @param sellerId
     * @return
     */
    Seller findSellerDetialInfo(String sellerId);

    /**
     * 更新商家信息
     * @param seller
     * @return
     */
    Integer updateSeller(Seller seller);

    /**
     * 查询商家账号信息
     * @param sellerId
     * @return
     */
    Seller findCountInfo(String sellerId);
}
